File: src/main/scala-graphql/io/github/graphql/model/ProfileOwnerTO.scala
package io.github.graphql.model


@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:00+0800"
)
trait ProfileOwnerTO {

    val email: String

    @javax.validation.constraints.NotNull
    val id: String

    @javax.validation.constraints.NotNull
    val itemShowcase: ProfileItemShowcaseTO

    val location: String

    @javax.validation.constraints.NotNull
    val login: String

    val name: String

    val pinnedItemsRemaining: Int

    val viewerCanChangePinnedItems: Boolean

    val websiteUrl: String

}

File: src/main/kotlin-graphql/io/github/graphql/kotlin/model/ProfileOwnerTO.kt
package io.github.graphql.kotlin.model


@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-07T19:48:13+0800"
)
interface ProfileOwnerTO {

    val email: String?

    val id: String

    val itemShowcase: ProfileItemShowcaseTO

    val location: String?

    val login: String

    val name: String?

    val pinnedItemsRemaining: Int

    val viewerCanChangePinnedItems: Boolean

    val websiteUrl: String?

}
